lib/pbench/server/cache_manager.py

```diff
@@ -893,6 +898,7 @@ def create(self, controller_name: str, tarfile: Path) -> Tarball:
             controller = Controller.create(controller_name, self.options, self.logger)
             self.controllers[controller_name] = controller
         tarball = controller.create_tarball(tarfile)
+        tarball.metadata = metadata
```

Comment:
You could probably get rid of the local `tarball` variable here without annoying `black` too badly ...

Reply:
How?

Reply:
Yeah, never mind.